CMakeLists.txt

```diff
@@ -119,33 +130,23 @@ if (MSVC)
     set(OPENSSL_USE_STATIC_LIBS true)
     set(OPENSSL_MSVC_STATIC_RT true)
 
-    if (ZEEK_STANDALONE)
-        include(${CMAKE_SOURCE_DIR}/cmake/conan.cmake)
-        conan_cmake_autodetect(settings)
-        # Install packages from conanfile
-        conan_cmake_install(PATH_OR_REFERENCE ${CMAKE_SOURCE_DIR}/ci/windows/conanfile_windows.txt
-                            BUILD missing SETTINGS ${settings})
-    endif ()
-
-    # Set LibPCAP to point to libpcap binaries.
+    # Set PCAP_ROOT_DIR to point at the installation from vcpkg. A later call
```

Comment:
The install section of the docs currently spells out chocolatey instructions. Looks like we'll need to update those, right? I'm not fully clear on how they relate.

Reply:
We still need the libraries that are installed by chocolatey. You just don't need to install conan anymore.